js/src/ui/AccountCard/accountCard.js

```diff
@@ -88,6 +93,7 @@ export default class AccountCard extends Component {
             className={ styles.balance }
             showOnlyEth
           />
+          { (password) ? this.renderPassword() : null }
```

Comment:
I was thinking about this too. Should really just do children, as I could see in the future using accountCards in this way.